.golangci.yml

```diff
@@ -19,6 +19,10 @@ linters:
     - unparam
     - whitespace
     - wsl
+linters-settings:
+  gosec:
+    excludes:
+      - G204
```

Comment:
Why do we need to exclude G204? What has changed?

Reply:
that's for `main_test.go` which we aren't linting currently but hopefully we can fix that in the future.